Description: Add Keywords to desktop file
Author: Jörg Frings-Fürst <debian@jff-webhosting.net>
Forwarded: not-needed
Last-Update: 2015-09-29
--
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/Index: trunk/remmina/desktop/remmina.desktop
===================================================================
Index: trunk/remmina/desktop/remmina.desktop.in
===================================================================
--- trunk.orig/remmina/desktop/remmina.desktop.in
+++ trunk/remmina/desktop/remmina.desktop.in
@@ -72,6 +72,7 @@ Terminal=false
 Type=Application
 Categories=GTK;GNOME;X-GNOME-NetworkSettings;Network;
 Actions=Profile;Tray;Quit;
+Keywords=remote desktop;rdp;vnc;nx;ssh;
 
 [Desktop Action Profile]
 Name=Create a New Connection Profile
